src/cclib/parser/data.py

```diff
@@ -298,6 +300,10 @@ def writexyz(self, filename=None):
         """Write parsed attributes to an XML file."""
         return self.write(filename=filename, outputtype='xyz')
 
+    @property
+    def electrons(self):

```

Comment:
Can we call it `nelectrons` to follow the same convention as `nmo`, `nbasis`, etc., ?